src/index.js

```diff
@@ -18,7 +18,7 @@ function main() {
       id: 1,
       classification: name,
       width: 300,
-      height: 100,
+      height: 115,
```

Comment:
Would be nice if this was automatically calculated but I didn't want to further complicate the PR.

Reply:
I think that can be arranged :)

Reply:
One issue is that I need the `window` in scope for the add button's click event. Because of this, I don't believe I can define the widgets prior to defining the window, and I'd need to know how big to make the window from the widgets. So I'm not entirely sure how one would do this.

Reply:
Yeah, you'd have to build a wrapper that invokes `ui.openWindow` after the `Document` has been initialized and all widgets have been determined.

Reply:
I actually think there's a way to do this, by using `ui.getWindow()` and passing in the classification "Benchwarmer". Will make a note to do this in the future.